astro/src/layouts/EscapeRoomLayout.astro

```diff
@@ -0,0 +1,456 @@
+          </p>
+        </div>
+      </div>
+    </footer>
+
+    <script src="../scripts/headings.js"></script>
+    <script src="../scripts/body-no-scroll.js"></script>
+    <script>
+      // Simple toggle for navbar
+      document.addEventListener('DOMContentLoaded', function() {
+        const toggler = document.getElementById('navToggler');
+        const menu = document.getElementById('navbarNav');
+        
+        if (toggler && menu) {
+          // Toggle menu on button click
+          toggler.addEventListener('click', function(e) {
+            e.stopPropagation();
+            const isExpanded = toggler.getAttribute('aria-expanded') === 'true';
+            toggler.setAttribute('aria-expanded', !isExpanded);
+            menu.classList.toggle('show');
+          });
+          
+          // Close menu when clicking elsewhere
+          document.addEventListener('click', function(e) {
+            if (!menu.contains(e.target) && !toggler.contains(e.target) && menu.classList.contains('show')) {
+              menu.classList.remove('show');
+              toggler.setAttribute('aria-expanded', 'false');
+            }
+          });
+          
+          // Close menu when clicking a link
+          const links = menu.querySelectorAll('.nav-link');
+          links.forEach(link => {
+            link.addEventListener('click', function() {
+              menu.classList.remove('show');
+              toggler.setAttribute('aria-expanded', 'false');
+            });
+          });
+        }
+      });
+    </script>
+    {
+      import.meta.env.PROD && (
+        <script
+          is:inline
+          defer
+          data-domain="accessiblecommunity.org"
+          src="https://plausible.io/js/script.js"
+        />
+      )
+    }
+  </body>
+</html>
+
+<style>
+  /* Custom Header Styles */
+  .escape-room-header {
+    background: linear-gradient(135deg, var(--bs-primary) 0%, #2c3e50 100%);
+    padding: 1rem 0;
+    box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.1);
+    position: relative;
+    z-index: 1000;
+  }
+
+  .escape-room-header .navbar {
+    padding: 0;
+    background: transparent;
+  }
+
+  .escape-room-header .navbar-brand {
+    color: white;
+    font-size: 1.5rem;
+    display: flex;
+    align-items: center;
+    gap: 0.75rem;
+    margin-right: 1rem;
+  }
+
+  .brand-text {
+    display: inline-block;
+    line-height: 1.2;
+  }
+  
+  .brand-text span {
+    display: inline;
+  }
+  
+  .brand-text .accessible::after {
+    content: " ";
+  }
+
+  .logo-container {
+    background: white;
+    border-radius: 0.5rem;
+    padding: 0.25rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.2);
+    flex-shrink: 0;
+  }
+
+  .logo-image {
+    width: 2.5rem;
+    height: 2.5rem;
+    object-fit: contain;
+  }
+
+  .escape-room-header .navbar-brand:hover .logo-container {
```

Comment:
I don't think we want the transition scale on hover. @brian-montgomery any thoughts?